content/en/docs/home/contribute/localization.md

```diff
@@ -8,83 +8,136 @@ approvers:
+The Kubernetes documentation for all languages is built from the [kubernetes/website](https://github.com/kubernetes/website) repository on GitHub. Most day-to-work work on translations, however, happens in separate translation repositories. Changes to those repositories are then [periodically](#upstream-contributions) synced to the main kubernetes/website repository via [pull request](../create-pull-request).
 
-### Basis for localizations
+Work on the Chinese translation, for example, happens in the [kubernetes/kubernetes-docs-zh](https://github.com/kubernetes/kubernetes-docs-zh) repository.
 
-Localizations must source from the English files for the [most recent major release](https://kubernetes.io/docs/home/supported-doc-versions/#current-version).
+{{< note >}}
+**Note**: For an example localization-related [pull request](../create-pull-request), see [this pull request](https://github.com/kubernetes/website/pull/8636) to the [Kubernetes website repo](https://github.com/kubernetes/website) adding Korean localization to the Kubernetes docs.
+{{< /note >}}
 
-**Note:** To find the most recent release's documentation source files:
-1. Navigate to https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+## Source Files
 
-    For example, the branch for Kubernetes v1.9 docs is `release-1.9`.
+Localizations must use English files from the most recent major release as sources. To find the most recent release's documentation source files:
 
-{{< note >}}
-Source files reside in the `/docs/` directory.
-{{< /note >}}
+1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
+2. Select the `release-1.X` branch for the most recent version, which is currently **{{< latest-version >}}**, making the most recent release branch [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
 
-### Repository
+## Getting started
 
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-zh](https://github.com/kubernetes/kubernetes-docs-zh).
+In order to add a new localization of the Kubernetes documentation, you'll need to make a few modifications to the site's [configuration](#configuration) and [directory structure](#new-directory), and then you can get to work [translating documents](#translating-documents)!
+
+To get started, clone the website repo and `cd` into it:
+
+```shell
+git clone https://github.com/kubernetes/website
+cd website
+git checkout {{< release-branch >}}
+```
 
 {{< note >}}
-**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+**Note**: There is currently no German translation of the Kubernetes documentation, but you're welcome to create one using the instructions outlined here!
```

Comment:
I'd remove this note and replace the related content on line 60 with a newline/this comment.

Reply:
Good catch! Done